src/api/stats/arbitrum/getRipaeApys.js

```diff
@@ -0,0 +1,120 @@
+const BigNumber = require('bignumber.js');
+const { MultiCall } = require('eth-multicall');
+const { arbitrumWeb3: web3, multicallAddress } = require('../../../utils/web3');
+
+const MasterChef = require('../../../abis/arbitrum/RipaeRewardPool.json');
+const ERC20 = require('../../../abis/ERC20.json');
+const fetchPrice = require('../../../utils/fetchPrice');
+const pools = require('../../../data/arbitrum/ripaeLpPools.json');
+const { BASE_HPY, ARBITRUM_CHAIN_ID } = require('../../../constants');
+const { getTradingFeeApr } = require('../../../utils/getTradingFeeApr');
+import { getFarmWithTradingFeesApy } from '../../../utils/getFarmWithTradingFeesApy';
+const { sushiArbitrumClient } = require('../../../apollo/client');
+const { compound } = require('../../../utils/compound');
+import { getContract, getContractWithProvider } from '../../../utils/contractHelper';
+
+const masterchef = '0x4d1D896FD501788d8605f672AD72fC05Fe5ab311';
```

Comment:
please change to getCommonMasterChef (bsc or fantom version), i see `paePerSecond` function.
also remove json abi